catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -753,6 +762,37 @@ public void setShouldConvertToBBox(boolean shouldConvertToBBox) {
     this.shouldConvertToBBox = shouldConvertToBBox;
   }
 
+  /**
+   * Get the number of vertices an approximation polygon will have when converting a multi
+   * point-radius search to a multi-polygon search.
+   */
+  public int getNumMultiPointRadiusVertices() {
+    return numMultiPointRadiusVertices;
+  }
+
+  /**
+   * Sets the number of vertices to use when approximating a polygon to fit to a multi point-radius
+   * search.
+   */
+  public void setNumMultiPointRadiusVertices(int numMultiPointRadiusVertices) {
+    this.numMultiPointRadiusVertices = numMultiPointRadiusVertices;
+  }
+
+  /**
+   * Get the distance tolerance value used for simplification of circular geometries.
+   */
+  public int getDistanceTolerance() {
+    return distanceTolerance;
+  }
+
+  /**
+   * Sets the distance tolerance value used for simplification of circular geometries.
+   */
+  public void setDistanceTolerance(int distanceTolerance) {
+    this.distanceTolerance =
```

Comment:
We may want to log that we are using the minimum because the admin entered a tolerance too low